src/Platform.Handlers/samples/Sample.Droid/MainActivity.cs

```diff
@@ -27,8 +30,68 @@ protected override void OnCreate(Bundle savedInstanceState)
 
 			_page = FindViewById<ViewGroup>(Resource.Id.pageLayout);
 
-			var app = new MyApp();
-			Add(app.CreateView());
+
+#if __REGISTRAR__
+			UseRegistrar();
+			Add(GetContentView());
+#else
+			var (host,app) = App.CreateDefaultBuilder()
```

Comment:
I left a comment above about the App but I don't think we'll want to perform all of this inside the Activity because in theory we'll want multi-window/multi-activity support so the App should exist at a level above the activity. 

But maybe this works for now until we define that?